src/main/webapp/app/exam/manage/exercise-groups/programming-exercise-cell/programming-exercise-group-cell.component.html

```diff
@@ -84,12 +84,13 @@
     @if (displayEditorModus) {
         <div>
             <div class="d-flex justify-content-between">
-                {{ 'artemisApp.programmingExercise.offlineIde' | artemisTranslate }}
-                : {{ programmingExercise.allowOfflineIde || false }}
+                <span jhiTranslate="'artemisApp.programmingExercise.offlineIde'">: {{ programmingExercise.allowOfflineIde || false }}</span>
             </div>
             <div class="d-flex justify-content-between">
-                {{ 'artemisApp.programmingExercise.onlineEditor' | artemisTranslate }}
-                : {{ programmingExercise.allowOnlineEditor || false }}
+                <span jhiTranslate="'artemisApp.programmingExercise.onlineEditor'">: {{ programmingExercise.allowOnlineEditor || false }}</span>
+            </div>
+            <div class="d-flex justify-content-between">
```

Comment:
There is no guard to only show this option if the `theia` profile is enabled on the server. Does it still make sense to show the Online-IDE-related options when `theia` is not active on the server?

(general question for all client code changes)

Reply:
Very fair question, I already was thinking about that but couldn't find a good solution yet. On the one hand, I think it would be cleaner to guard the whole functionality for now (and for universities that don't want to use Theia). But on the other hand would such a guard clutter up the whole code and introduce dependencies to the profiles all over the exercise workflow (my changes)..